### DISCUSSION TITLE
Getting correct timestamps from model stt_en_fastconformer_hybrid_large_pc

### DISCUSSION THREAD

**Discussion by Khimer** [NVIDIA-NeMo/NeMo#6964] [Q&A] [answered]
Hello!
I am trying to get timestamps from a model (stt_en_fastconformer_hybrid_large_pc) following the instructions from https://colab.research.google.com/github/NVIDIA/NeMo/blob/main/tutorials/asr/Buffered_Transducer_Inference.ipynb#scrollTo=NBSX6HS44pBR but unfortunately I encounter the fact that the timestamps do not correspond to reality. For audio 25 seconds long, the timestamp of the last token is at 18 seconds, even though that token is at the end of the audio.
I tried to change model_stride == 8, but in this case, the timestamp of the last token is outside the audio.
Should we use other parameters for the fastconformer_hybrid model?
Thank you!

**Comment by titu1994** [comment, marked as answer]
You do have to use 8 as model stride. Can you paste the full hypothesis in a file ? It is not really possible to go out of bounds in audio dimension as T is defined by the maximum audio duration T * 100 / 8.
